app/frontend/styles/publications/_itt_grid.scss

```diff
@@ -0,0 +1,16 @@
+.govuk-grid-row:has(.itt-grid) {
+  margin-right: 0;
+}
```

Comment:
If `has` isn't supported, there will just be no margin on the right side of the itt grid, as is the case now.
This rule makes it look a bit better.